pootle/apps/pootle_store/utils.py

```diff
@@ -212,17 +203,26 @@ def accept_suggestion(self, suggestion, update_unit):
         unit.target = suggestion.target
         if unit.state == FUZZY:
             unit.state = TRANSLATED
-
+        unit.submitted_by = suggestion.user
+        unit.submitted_on = current_time
+        unit.reviewed_by = self.reviewer
+        unit.reviewed_on = unit.submitted_on
+        unit.save(
+            submitted_by=suggestion.user,
```

Comment:
yes - because its used to set unit change/unit_source objects - the above lines will be removed pretty soon